test/format/util.c

```diff
@@ -348,6 +348,12 @@ path_setup(const char *home)
 	testutil_check(__wt_snprintf(
 	    g.home_stats, len, "%s/%s", g.home, "stats"));
 
+	/* Tree dump file. */
+	len = strlen(g.home) + strlen("pagedump") + 2;
+	g.home_pagedump = dmalloc(len);
+	testutil_check(__wt_snprintf(
+	    g.home_pagedump, len, "%s/%s", g.home, "pagedump"));
+
```

Comment:
Not sure the comment makes sense any more.

Reply:
Fixed